### DISCUSSION TITLE
Ability to exclude all repos that are a fork of a specific repo

### DISCUSSION THREAD

**Discussion by simonw** [community/community#44952] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Here's an example of a search I just tried:

    register_output_renderer -repo:simonw/datasette

I get a whole lot of results for this search that are from forks of my `simonw/datasette` repository. I'd like to be able to exclude not just that repository but also anything that's a fork of that repository.

**Comment by jge162** [comment]
try this @simonw to exclude forks of a specific repository in GitHub search by using the "is:fork" filter. To exclude all forks of the simonw/datasette repository, you can use the following search query: "repo:simonw/datasette is:public -is:fork". This will return results for the simonw/datasette repository that are not forks.

**Comment by look** [comment, marked as answer]
Thanks for the feedback @simonw. This is a feature I'd like to build, including allowing you to search within a network (something like `network:torvalds/linux` or `is:network(torvalds/linux)`). We are prioritizing scaling the system right now, but hopefully we can address this feature request when that's done.

**Comment by jsoref** [comment]
Fwiw, today you can exclude all forks:
https://github.com/search?q=register_output_renderer+NOT+repo%3Asimonw%2Fdatasette+NOT+is%3Afork&type=code

It is not _technically_ the same as what you want, but at least for most cases, it's probably good enough.
